### DISCUSSION TITLE
世界における接触通知アプリの状況

### DISCUSSION THREAD

**Discussion by keiji** [cocoa-mhlw/cocoa#1000] [General]
世界における接触通知アプリで非アクティブ化・運用を停止した事例があるので状況をまとめておきます。

## 世界の接触通知アプリ一覧（Google Play
https://developers.google.com/android/exposure-notifications/apps

## 非アクティブ化（一時停止
 * smitte|stop（デンマーク・ノルウェー） - https://smittestop.dk/en 
     * Now you can stop self-isolating sooner - https://www.sst.dk/en/English/News/2022/Now-you-can-stop-self-isolating-sooner
 * CoronaMelder（オランダ） - https://coronamelder.nl/en/
     * CoronaMelder is temporarily suspended - https://coronamelder.nl/en/
 * COVID Alert（カナダ） - https://www.canada.ca/en/public-health/services/diseases/coronavirus-disease-covid-19/covid-alert.html
     * Canada sunsets its COVID Alert app based on the iOS and Android exposure notification API - https://techcrunch.com/2022/06/17/canada-sunsets-its-covid-alert-app-based-on-the-ios-and-android-exposure-notification-api/amp/
 * STAYAWAY COVID（ポルトガル） - https://stayawaycovid.pt/en 
 * COVID Alert CT（アメリカ コネチカット州） - https://play.google.com/store/apps/details?id=gov.ct.covid19.exposurenotifications
 * COVID Alert DE（アメリカ デラウェア州） - https://coronavirus.delaware.gov/covidalert/
 * Stopp Corona（オーストリア） - https://www-stopp--corona-at.translate.goog/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=ja&_x_tr_pto=wapp
     * https://github.com/austrianredcross/stopp-corona-android
 * CovTracer-EN（キプロス）- https://covtracer.dmrid.gov.cy/dmrid/covtracer/covtracer.nsf/home_en/home_en?opendocument
 * eRouška（チェコ共和国） - https://erouska-cz.translate.goog/caste-dotazy?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=ja&_x_tr_pto=wapp
     * https://github.com/covid19cz/erouska-android
 * HOIA（エストニア） - https://www.tehik.ee/en/uudis/may-hoia-app-will-be-deactivated
 * Koronavilkku（フィンランド） - https://koronavilkku-fi.translate.goog/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=ja&_x_tr_pto=wapp
 * COVID Alert Malta（マルタ共和国）
 * SwissCovid（スイス） - https://www.bag.admin.ch/bag/en/home/krankheiten/ausbrueche-epidemien-pandemien/aktuelle-ausbrueche-epidemien/novel-cov/swisscovid-app-und-contact-tracing.html
 * MI COVID Alert（アメリカ ミシガン州） - https://www.michigan.gov/coronavirus/contain-covid/covidalert

----

https://play.google.com/store/apps/details?id=gov.ct.covid19.exposurenotifications

smitte|stopは、アプリを立ち上げると "smitte|stop is not active and is not currently maintained. You can delete the app. If the health authorities decide to provide the app again. it will be updated" という画面を表示する。
CoronaMelderも同様に、アプリを立ち上げると"CoronaMelder is closed"という画面を表示する。

![Screen Shot 2022-05-07 at 1 02 05](https://user-images.githubusercontent.com/932136/167170586-7c6d3e09-fdd1-4993-993f-54bfa7581ec9.png)

![Screen Shot 2022-05-07 at 1 02 23](https://user-images.githubusercontent.com/932136/167170609-0906d645-e42b-4598-930a-53eaff31c116.png)

両アプリともに、国の感染症対策の方針変更。特に接触時の自己隔離措置の撤廃という変化に対応しての非アクティブ化という流れの様子。

**Comment by keiji** [comment]
カナダのCOVID Alertアプリが終了。

> COVID Alert is now retired
> You can delete the app. Thank you for doing your part to help limit the spread of COVID-19.

https://www.canada.ca/en/public-health/services/diseases/coronavirus-disease-covid-19/covid-alert.html

COVID Alertアプリの終了は、前の2国と異なり、国の感染症対策の方針変更に基づかない、どちらかというと陽性登録者数が伸びず、ExposureNotificationの効果が見えないというニュアンスが強そう。

Canada sunsets its COVID Alert app based on the iOS and Android exposure notification API
https://techcrunch.com/2022/06/17/canada-sunsets-its-covid-alert-app-based-on-the-ios-and-android-exposure-notification-api/amp/

**Comment by keiji** [comment]
ポルトガルのSTAYAWAY COVIDアプリが終了

> Notwithstanding that the end of COVID-19 has not been declared yet, the positive epidemiological evolution of the disease in Portugal supports the interruption of the STAYAWAY COVID system operation.

https://stayawaycovid.pt/en

" COVID-19は終結していませんが、疫学的な進展はSTAYAWAY COVIDの運用を中断することを支持しています。"

疫学的な変化が感染症対策の方針変更に繋がったと考えてよさそう。

**Comment by keiji** [comment]
アメリカ コネチカット州のCOVID Alert CTが運用終了（？）

https://play.google.com/store/apps/details?id=gov.ct.covid19.exposurenotifications

Google Playでは「リソースが見つからない」と表示される。現状や終了に関するアナウンスは見つからない。

**Comment by keiji** [comment]
アメリカ・デラウェア州のCOVID Alert DEが運用終了（？）

https://coronavirus.delaware.gov/covidalert/

Google Playでは「リソースが見つからない」と表示される。現状や終了に関するアナウンスは見つからない。

コネチカット州もそうだけど、COVID Alertシリーズは停止している可能性がある。

**Comment by keiji** [comment]
オーストリアのStopp Coronaアプリが運用終了

https://www-stopp--corona-at.translate.goog/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=ja&_x_tr_pto=wapp

https://github.com/austrianredcross/stopp-corona-android

**Reply by daisuke-nogami** [reply]
https://www.stopp-corona.at/
（フィンランドとかスイスのような多言語国家は英語版を標準で作ってくれるけれどオーストリアはドイツ語だけだな…）

**Comment by keiji** [comment]
キプロスのCovTracer-ENアプリが運用終了

https://covtracer.dmrid.gov.cy/dmrid/covtracer/covtracer.nsf/home_en/home_en?opendocument

**Comment by keiji** [comment]
チェコ共和国のeRouškaアプリが運用終了。
（これ運用終了が2021年10月と結構前だけど、ぼくが参照している[ENアプリ一覧](https://developers.google.com/android/exposure-notifications/apps)って、運用終了したアプリは行毎削除される仕組みなんだな……）

https://erouska-cz.translate.goog/caste-dotazy?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=ja&_x_tr_pto=wapp

https://github.com/covid19cz/erouska-android

**Comment by keiji** [comment]
エストニアのHOIAアプリの運用終了を確認。

https://www.tehik.ee/en/uudis/may-hoia-app-will-be-deactivated

**Comment by keiji** [comment]
フィンランドのKoronavilkkuアプリの運用終了を確認。

https://koronavilkku-fi.translate.goog/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=ja&_x_tr_pto=wapp

**Reply by daisuke-nogami** [reply]
URLはこっちの方がオフィシャルかも
https://koronavilkku.fi/en/

**Comment by keiji** [comment]
マルタ共和国のCOVID Alert Maltaの運用終了を確認。
COVID Alertシリーズなのでコネチカット州とデラウェア州の仲間かも。

ウェブサイトはあるけど、証明書が8月3日で期限切れと言われる…
https://covidalert.gov.mt/

**Comment by keiji** [comment]
スイスのSwissCovidの運用停止（Temporary suspension）を確認。

https://www.bag.admin.ch/bag/en/home/krankheiten/ausbrueche-epidemien-pandemien/aktuelle-ausbrueche-epidemien/novel-cov/swisscovid-app-und-contact-tracing.html

> Now that the isolation requirement has been lifted, the conditions for the effective continued operation of the SwissCovid app are no longer in place, at least temporarily.

隔離要件の閾値の上昇したので、みたいなニュアンスだろうか。

**Reply by daisuke-nogami** [reply]
ここの lifted は「解除された」 https://eow.alc.co.jp/search?q=lift という意味ですね。
隔離するルールがなくなったのでSwissCovidの意味がなくなりました、だから一時的に止めました、と説明されていますね。

https://www.bag.admin.ch/bag/en/home/krankheiten/ausbrueche-epidemien-pandemien/aktuelle-ausbrueche-epidemien/novel-cov/massnahmen-des-bundes.html
> National rules
> There are currently no national measures in place.

https://twitter.com/swissinfo_jp/status/1509176006191501324
> スイス連邦政府は30日、公共交通機関・医療施設でのマスク着用義務と、陽性者の隔離義務を予定通り3月末で解除すると発表。これで全ての感染対策がなくなり、スイス社会は完全に正常化することになりました　https://www.swissinfo.ch/jpn/society/%E6%96%B0%E5%9E%8B%E3%82%B3%E3%83%AD%E3%83%8A%E3%82%A6%E3%82%A4%E3%83%AB%E3%82%B9-%E3%82%B9%E3%82%A4%E3%82%B9-%E6%9C%80%E6%96%B0%E6%83%85%E5%A0%B1-45591162

**Reply by parasite2006** [reply]
なるほど、感染確認者と接触した人に自主隔離を要請する規則が撤廃されることになると保健当局は接触確認アプリの運用停止を決定するのですね。
たとえば人口133万人のエストニア
https://www.mofa.go.jp/mofaj/area/estonia/data.html#section1
は、1日あたりの新規感染者数（7日間移動平均）が2022年2月5日（第6波のピーク時）の6861人から2022年8月31日現在の100人台まで低下。
https://coronavirus.jhu.edu/region/estonia
日本の人口1億2000万人に換算すると、全国の新規感染者数1日あたり1万人台（2021年夏の第5波のピーク時の約半分）に相当するレベルですが、よその国ではこの状況で自主隔離要請をやめるのかと言いたい気もする。

**Reply by daisuke-nogami** [reply]
たまたまスイスが、ENの運用と、強制隔離（スイスの場合は自主ではなく隔離のルールがあったかと思います）のルールを連動させて説明していただけなので、全ての国でイコールという分けではないと思います。

感染者数の多寡と隔離のルールも必ずしも連動しません。
そのあたりの議論は、西浦博先生のインタビューを踏まえると分かるところですが…

https://www.buzzfeed.com/jp/naokoiwanaga/covid-19-nishiura-20220708-3
> 欧州や米国で緩和が起こると、それらの国では自然に感染する人の数が桁違いに多くなりました。感染者数が多くなるということは、実はウイルスの進化が起こる（変異が起こる）チャンスが多くなる、ということに繋がります。
> ウイルスが多くの人の体に侵入して増殖し、その都度、変異の機会があるためです。結果として、オミクロンの派生型が現れてきたように、既存の免疫を回避するウイルスが現れ、それが他を置き換えて流行を起こしてしまうリスクが日に日に高くなりました。
(中略)
> ワクチン開発の速度が、緩和によるウイルスの進化速度に追い付いていない以上、どこかで自然感染による免疫を人口レベルで獲得する方針に一気に舵を切らざるを得ない可能性が高いわけです。
> そういうことを考えれば、英国は2022年1月に大規模な緩和を決断したわけですが、本来、そういう選択は国際的に協調しつつ実施するべきものです。日本を含む国際社会は、独断で突き進んだ英国に対して苦言を呈さねばならなかったものと考えています。
> いずれにしても、みんなの免疫の状況を見ながら、予防接種に頼る方法から自然免疫獲得をも見越して、死亡者数を最少にする方針へ切り替えるタイミングを科学的に測れないか、研究者として考えています。

https://www.buzzfeed.com/jp/naokoiwanaga/covid-19-nishiura-20220819-1
> 「エピデミック」は平時で期待される感染者数を異常に上回っている状態です。集団の中で本来あるはずのない流行が起きていると、それだけで「エピデミック」と呼びます。
> 「パンデミック」は「エピデミック」が国境を超えて、国際的に広がった状態です。世界的な流行です。
> 「エンデミック」は「風土病的流行」と書かれることがあります。病原体が特定の人口に根付いて、常在している状態です。
（略）
> 今までエピデミックの状態で、「ハンマー＆ダンス」の対応を取ってきました。つまり、流行が一気に拡大しようとする時点で、感染拡大を抑える強い対策を打って叩く（ハンマー）。一気に新規感染者数を下げたら、また対策を緩めて過ごす（ダンス）。その繰り返しです。
> それはエピデミックの時期だけに行うことで、エンデミックになると人口内で感染が常在することを許す状況になります。
（略）
> エンデミック化は少なくとも常に少なくとも1〜2%の感染者がいることを受け入れるということです。

https://www.buzzfeed.com/jp/naokoiwanaga/covid-19-nishiura-20220819-2
> 感染レベルが高くなっているなかで、皆さんのムードとしてはこのまま進んでいくのかなと思っていますよね。これを超えると、集団に免疫ができて、その後流行はしばらく起こらないと誤解しているかもしれません。それは大きな間違いです。
> この感染症は何度も流行のループを繰り返し形成して、獲得された免疫から逃げ、次の流行を起こすことが今までの経験でわかっています。今回と同じようなレベルの流行がまた起こります。むしろ社会はどんどん解放されているので、今後の流行の方がこれまでより大きくなる状況がしばらく続くと思います。

**Comment by keiji** [comment]
アメリカ ミシガン州のMI COVID Alertの運用終了を確認。

https://www.michigan.gov/coronavirus/contain-covid/covidalert

> As of July 1, 2022, the MI COVID Alert app will no longer be in service. This change is part of the ongoing work to determine the state’s most current needs as we respond to COVID-19. The Michigan Department of Health and Human Services continues to provide COVID-19 information and resources to help you protect yourself and your family.
